storage/libradb/src/state_store/state_store_test.rs

```diff
@@ -232,7 +232,7 @@ proptest! {
     #![proptest_config(ProptestConfig::with_cases(10))]
 
     #[test]
-    fn test_iter_accounts(
+    fn test_get_account_iter(
```

Comment:
not sure if we want to move this test to another file since the method is no longer in `state_store`.

Reply:
I thought about that, but the machinery is already so convenient here and under the hood it is testing the state store..